docs/03-policies/prodev.md

```diff
@@ -0,0 +1,102 @@
+| 2016  | 963.5                          | 28                            | 34.4                                          |
+| 2017* | 1620                           | 35                            | 46.3                                          |
+ 
+_*As of Nov 21, 2017. All data was pulled from Harvest. [Link to data & calculations here.](https://docs.google.com/spreadsheets/d/12aWOg9I_nbolpaA9j4vbsv5ABZW2hYXfjkcRwRpTs1I/edit#gid=2143420373)_
+ 
+ ## Prodev Sprints
+ 
+ In order to provide some additional structure, support and focus to the prodev efforts of individuals, 1 month long prodev sprints will be held on a regular basis. During these sprints, participants will attend scrum meetings 2 to 3 times per week to talk about their prodev progress being made and connect with others pursuing their prodev goals at the same time. The sprint will also consist of a demo or review meeting, so that team members can share what they have accomplished or learned during the sprint. A retrospective will also take place so that the prodev sprint process can be continuously improved upon. 
+
+The total time commitment for participating in a prodev sprint will vary widely depending on what topic and intensity level each participant chooses to pursue, but the prodev sprint is not considered a full time project.
+
+Assignment to a prodev sprint is voluntary, and team members are encouraged to express their desire to participate to the prodev Practice Lead, Jason Heaster. Efforts will be made to group people with similar prodev goals or certification requirements. Once assigned to a sprint, participants’ project assignment schedules will be adjusted in Forecast to allocate time for the prodev sprint during that month.
+
+## Making a Plan
+
+Previously, CivicActions had a professional development process that included regular management review of prodev plans created on trello boards. While this isn’t a required practice anymore, you are still encouraged to take responsibility for your own professional development by creating a prodev plan and setting goals for yourself. If you would like help with structuring a plan, use [this trello board template](https://trello.com/b/p7FOD0Ju/template-professional-development-and-community-participation) as a guide or ask the prodev Practice Lead for help.
+
+## Asking a Mentor, Coach, or Peer to Help
+
+For some people, getting input from others is an important part of the professional development process. You are encouraged to seek out other CivicActioners who you think could help you achieve your prodev goals. This may be a manager, a peer, or any person who is good at something you’d like to learn.
+
+Also, if you’re asked by someone else to assist with their prodev, please make time to help them out!
+
+## Communication
+
+We have a [#prodev](https://civicactions.slack.com/messages/prodev) Slack channel for prodev-related communication.
+
+## Prodev FAQ's
+
+Q: How much do I get to spend on prodev each year?  
```

Comment:
Are we still requesting management approval before purchasing? I didn't see that in the process noted above